src/gobexport/__main__.py

```diff
@@ -34,6 +42,19 @@ def _get_filename(name):
 
 
 def export_collection(host, catalog, collection, file_name):
+    # Extra variables for logging, generate them since we do not get them from workflow yet
+    global extra_log_kwargs
+    start_timestamp = int(datetime.datetime.now().replace(microsecond=0).timestamp())
+    source = 'GOB Objectstore'
```

Comment:
Isn,t the API the source and Objectstore the destination?

Reply:
Yes that's actually better, but at this moment we only have source as a field on the logs in the database. I think we need to redefine our logging model, because it's only focused on the import for now.